ethcore/res/ethereum/ellaism.json

```diff
@@ -28,7 +28,6 @@
 		"eip161abcTransition": "0x7fffffffffffffff",
 		"eip161dTransition": "0x7fffffffffffffff",
 		"eip155Transition": "0x0",
-		"wasmActivationTransition": 2000000,
```

Comment:
It seems you make the same change in https://github.com/vorot93/oe_playground/pull/40